hazelcast-client-new/src/test/java/com/hazelcast/client/map/ClientMapTest.java

```diff
@@ -226,6 +226,20 @@ public void testGetAllPutAll() {
     }
 
     @Test
+    public void testPutAllWithTooManyEntries() {
```

Comment:
What is the test case description here? Does this test really verify that the request is not randomly sent but to the specific partition Id? When sent to a random partition id, will the call "map.get(i)" fail?